fobi_custom/plugins/form_elements/fields/observational/age/forms.py

```diff
@@ -2,66 +2,58 @@
-                            "The Public Life Data Protocol "
-                            "suggests collecting age data at one "
-                            "of three levels of detail:<br /><br />"
-                            "<b>Basic</b><br />"
-                            "0-14<br />"
-                            "15-24<br />"
-                            "25-64<br />"
-                            "65+<br /><br />"
-                            "<b>Detailed</b><br />"
-                            "0-4<br />"
-                            "5-14<br />"
-                            "15-24<br />"
-                            "25-44<br />"
-                            "45-64<br />"
-                            "65-74<br />"
-                            "75+<br /><br />"
-                            "<b>Complex</b><br />"
-                            "0-4<br />"
-                            "5-9<br />"
-                            "10-14<br />"
-                            "15-17<br />"
-                            "18-24<br />"
-                            "25-34<br />"
-                            "35-44<br />"
-                            "45-54<br />"
-                            "55-64<br />"
-                            "65-74<br />"
-                            "75+<br /><br />")
+    detail_level = forms.ChoiceField(choices=AGE_TYPE_CHOICES,
+                                     help_text="Following the Public Life Data "
+                                     "Protocol, age data should be collected at "
+                                     "one of the following levels of detail:<br /><br />"
+                                     "<b>Basic</b><br />"
+                                     "0-14<br />"
+                                     "15-24<br />"
+                                     "25-64<br />"
+                                     "65+<br /><br />"
+                                     "<b>Detailed</b><br />"
+                                     "0-4<br />"
+                                     "5-14<br />"
+                                     "15-24<br />"
+                                     "25-44<br />"
+                                     "45-64<br />"
+                                     "65-74<br />"
+                                     "75+<br /><br />"
+                                     "<b>Complex</b><br />"
+                                     "0-4<br />"
+                                     "5-9<br />"
+                                     "10-14<br />"
+                                     "15-17<br />"
+                                     "18-24<br />"
+                                     "25-34<br />"
+                                     "35-44<br />"
+                                     "45-54<br />"
+                                     "55-64<br />"
+                                     "65-74<br />"
+                                     "75+<br /><br />")
```

Comment:
Are these choices initialized with the default value `0`? It occurs to me that in the case of `detailed` and `complex` groupings, the survey runner is probably not going to want to fill out every field with `0` that doesn't contain a category they saw.

Reply:
They're initialized with an empty box that currently saves as "None". I think there's value in keeping that input blank before the survey runner puts in any data, so they can tell the difference between a recorded 0 and a box they haven't looked at yet, but also that it might make sense to save those as 0 if there's no reason on the DAD end to distinguish those 2 cases.

Reply:
Good point. Let's punt on this and return to it once we have a better sense of data requirements for the DAD.